blocks/logic.js

```diff
@@ -231,7 +231,7 @@ Blockly.defineBlocksWithJsonArray([  // BEGIN JSON EXTRACT
         "name": "ELSE"
       }
     ],
-    "output": true,
+    "output": null,
```

Comment:
I don't follow this part of the change. Can you explain some more? If I'm looking at the right thing, it looks like before this was converted to json we did a setOutput(true).